recipes/flang-activation/activate.sh

```diff
@@ -0,0 +1,8 @@
+export _OLD_FC=$FC
+export _OLD_FFLAGS=$FFLAGS
+export _OLD_FORTRANFLAGS=$FORTRANFLAGS
```

Comment:
Let's use `_CONDA_BACKUP` instead of `_OLD` to match other compilers.

Reply:
Fine by me. GCC [uses](https://github.com/conda-forge/ctng-compiler-activation-feedstock/blob/e22827cd872402d807ae3452e23673d6d3e4b0b0/recipe/activate-gcc.sh#L45) `CONDA_BACKUP` without a leading underscore, so I'll follow that for consistency.